norfair/utils.py

```diff
@@ -18,7 +20,7 @@ def validate_points(points):
     return points
 
 
-def print_objects_as_table(tracked_objects):
+def print_objects_as_table(tracked_objects: List):
```

Comment:
Same as mentioned previously.

Reply:
Calling TrackedObject in utils.py causes import error.
You can see the error in the github actions at the following link.
https://github.com/tryolabs/norfair/runs/1152822142?check_suite_focus=true

Reply:
Yeah we gotta add `from .tracker import TrackedObjec` to utils.py. Dont worry i'll fix it.

Reply:
Turns out its harder than I thought as its triggering circular imports.